grunt-ngdocs-index.tmpl

```diff
@@ -89,30 +89,88 @@
   </script>
 </head>
 
-<body>
-  <header class="header">
-    <div class="navbar navbar-fixed-top">
-      <div class="navbar-inner">
-        <div class="container-fluid navbar-inverse">
-          <% if (image) { %>
-            <% if (imageLink) { %>
-              <a href="<%= imageLink %>">
-            <% } %>
-              <img class="pull-left" src="<%= image %>">
-            <% if (imageLink) { %>
+<body class="showcase">
+  <header>
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle" ng-click="isOpen = !isOpen">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <div class="navbar-brand">
+            <a id="logo" title="Angular Patternfly" href="#">
+              PATTERNFLY-<span class="secondary-logo">ANGULAR</span>
+            </a>
+          </div>
+        </div>
+        <div class="navbar-collapse navbar-collapse-pf-site collapse">
+          <ul id="menu-primary" class="nav navbar-nav navbar-right">
+            <li class="active">
+              <a href="#">Components</a>
+            </li>
+            <li>
+              <a href="https://github.com/patternfly/angular-patternfly">Github</a>
+            </li>
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                <i class="fa fa-th-large" aria-hidden="true">
+                  <span class="sr-only">PatternFly Showcases</span>
+                </i>
               </a>
-            <% } %>
-          <% } %>
-          <% if (title) {%>
-            <% if (imageLink) { %>
-              <img class="pull-left" src="<%= imageLink %>">
-            <% } %>
-            <a <% if (titleLink) { %> href="<%= titleLink %>" <% } %> class="navbar-brand"><%= title %></a>
-          <% } %>
-          <%= navContent %>
+              <ul class="dropdown-menu">
+                <li>
+                  <a href="https://rawgit.com/patternfly/patternfly-ng/master-dist/dist-demo/#/">PatternFly-ng</a>
+                </li>
+                <li>
+                  <a href="https://patternfly-webcomponents.github.io/components.html">PatternFly-WebComponents</a>
+                </li>
+                <li>
+                  <a href="https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html">PatternFly React</a>
```

Comment:
I see different formatting for the names

- Patternfly-ng
- Patternfly-WebComponents
- Patternfly React
- Patternfly CSS

These should either be the repo name:

- patternfly-ng
- patternfly-webcomponents
- patternfly-react
- patternfly-css

Or the titles:

- Patternfly NG
- Patternfly Web Components
- Patternfly React
- Patternfly CSS